internal/datastore/datastore.go

```diff
@@ -14,8 +14,9 @@ const Ellipsis = "..."
 
 // RevisionChanges represents the changes in a single transaction.
 type RevisionChanges struct {
-	Revision Revision
-	Changes  []*v0.RelationTupleUpdate
+	Revision            Revision
+	Changes             []*v0.RelationTupleUpdate
+	RelationshipUpdates []*v1.RelationshipUpdate
```

Comment:
Since `RelationTupleUpdate` objects are trivially convertible to `RelationshipUpdate` objects we should pick one and only one for the datastore interface.

Reply:
Ok, sounds good. I've undone these changes in favor of doing the conversion in the Watch implementation instead.